cmd/ytt/ytt.go

```diff
@@ -24,3 +27,12 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func enableExperimentalFeatures() {
+	if experiments, isSet := os.LookupEnv("YTTEXPERIMENTS"); isSet {
+		for _, experiment := range strings.Split(experiments, ",") {
+			feature.Flags().Enable(experiment)
+			fmt.Fprintf(os.Stderr, "Experimental feature %q enabled.\n", experiment)
```

Comment:
I am not sure on this, but we should validate with kapp-controller if writing to stderr somehow interferes with how they report a successful template step. (This may not be an issue at all, just covering our bases here)

Reply:
Great question!

I'll follow-up.

Reply:
It does not. 👍🏻 

https://kubernetes.slack.com/archives/CH8KCCKA5/p1652123416054149